packages/jest-test-result/tsconfig.json

```diff
@@ -6,6 +6,8 @@
   },
   "references": [
     {"path": "../jest-console"},
+    {"path": "../jest-coverage"},
```

Comment:
Um, am I missing something or the `jest-coverage` module is not used anymore?

Reply:
Yea, I missed it's under `@jest/coverage` 🤦‍♂

Reply:
It's used here https://github.com/facebook/jest/pull/8596/files#diff-1110b7804757285651cfaf9e02afdd2eR17